[rospy.client][INFO] 2021-11-24 14:13:59,610: init_node, name[/server_endpoint], pid[2489]
[xmlrpc][INFO] 2021-11-24 14:13:59,610: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-11-24 14:13:59,610: Started XML-RPC server [http://wng:37567/]
[rospy.init][INFO] 2021-11-24 14:13:59,610: ROS Slave URI: [http://wng:37567/]
[rospy.impl.masterslave][INFO] 2021-11-24 14:13:59,611: _ready: http://wng:37567/
[rospy.registration][INFO] 2021-11-24 14:13:59,611: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2021-11-24 14:13:59,611: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-11-24 14:13:59,711: registered with master
[rospy.rosout][INFO] 2021-11-24 14:13:59,711: initializing /rosout core topic
[rospy.rosout][INFO] 2021-11-24 14:13:59,712: connected to core topic /rosout
[rospy.simtime][INFO] 2021-11-24 14:13:59,712: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-11-24 14:13:59,716: Starting server on 192.168.1.146:10000
[rospy.internal][INFO] 2021-11-24 14:13:59,878: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-11-24 14:14:34,945: Connection from 192.168.1.146
[rosout][INFO] 2021-11-24 14:14:34,949: RegisterPublisher(execution_check, <class 'panda_move.msg._ExecutionCheck.ExecutionCheck'>) OK
[rospy.internal][INFO] 2021-11-24 14:14:35,000: topic[/execution_check] adding connection to [http://wng:37567/], count 0
[rospy.internal][INFO] 2021-11-24 14:14:35,000: topic[/execution_check] adding connection to [/server_endpoint], count 0
[rospy.core][INFO] 2021-11-24 14:47:25,087: signal_shutdown [signal-2]
[rospy.internal][INFO] 2021-11-24 14:47:25,090: topic[/execution_check] removing connection to /server_endpoint
[rospy.internal][ERROR] 2021-11-24 14:47:25,091: Unable to initiate TCP/IP socket to wng:35695 (http://wng:37567/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 600, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-11-24 14:47:25,091: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-11-24 14:47:25,091: topic[/execution_check] removing connection to http://wng:37567/
[rospy.impl.masterslave][INFO] 2021-11-24 14:47:25,092: signal-2
[rospy.core][INFO] 2021-11-24 14:47:25,207: signal_shutdown [atexit]
